.github/workflows/platform-builders.yml

```diff
@@ -0,0 +1,264 @@
+name: Build
+
+on:
+  push:
+    branches: ['**', '!master']
+  pull_request_target:
+    branches: ['master']
+    types:
+      - closed
+
+jobs:
+
+  alpine:
+    if: github.event_name == 'push' || github.event.pull_request.merged == true
```

Comment:
Is this still needed? The build should only run at this time based on the ```on:``` settings, no?

Reply:
What I'm trying for here is a combination of
1. Trigger the workflow when we push to any non-`master` branch, whether it's affiliated with a PR or not
2. Avoid triggering the same workflow a second time in the (common) case where there _is_ a PR for the branch
3. Trigger the workflow when the PR is merged to `master`

The `'push'` part of the conditional indeed seems redundant with what's in `on: push`. If the second trigger could be for `on: pull_request_target: types: merged`, then I think we could get rid of the entire `if:` line for each job. But my understanding (from [Stack Overflow](https://stackoverflow.com/questions/60710209/trigger-github-actions-only-when-pr-is-merged)) is that we can't say `merged` in the trigger, only `closed`, which means that in jobs we have to check conditionally which kind of `closed` it was. :-/